### PR TITLE
Show bold text for active set bonuses in Build Editor / Sets

### PR DESCRIPTION
**Pull request by khuusko**

### CODE REVIEW

**Comment by khuusko** [comment]
Whoops, accidentally tried to merge, unintentional.

File: src/components/GearCard.tsx [hunk 1]
```diff
@@ -45,146 +45,168 @@ const StyledTag = styled(Tag)`
...
       />
       <Description>
-        <div style={{ display: "flex", flexDirection: "column" }}>
-          {totalBonus(set).map(i => (
-            <span key={i}>
-              {i} pcs: {set && set[`bonus_item_${i}`]}
-            </span>
-          ))}
+        <div style={{ display: 'flex', flexDirection: 'column' }}>
+          {totalBonus(set).map(i => {
```
Comment (fewhnhouse) [inline src/components/GearCard.tsx]: rename variable i to something more declarative, like count

File: src/components/GearCard.tsx [hunk 2]
```diff
@@ -45,146 +45,168 @@ const StyledTag = styled(Tag)`
...
   width: 25px;
   height: 25px;
   margin-right: 5px;
-`;
+`
 
 interface IGearCard {
-  set: ISet;
+  set: ISet
+  bonuses?: Map<any, any>
```
Comment (fewhnhouse) [inline src/components/GearCard.tsx]: Your map should be of type <string, number>

File: src/components/GearCard.tsx [hunk 3]
```diff
@@ -45,146 +45,168 @@ const StyledTag = styled(Tag)`
...
+        return ''
     }
-  };
+  }
 
   return (
     <Container>
       <Title>
-        {gear.selectedSet ? gear.selectedSet.name : 'Set name'} {gear.weaponType ? <SmallTitle>{gear.weaponType}</SmallTitle> : ""}
+        {gear.selectedSet ? gear.selectedSet.name : 'Set name'}{' '}
```
Comment (fewhnhouse) [inline src/components/GearCard.tsx]: remove {' '}

File: src/pages/build/Sets/Sets.tsx [hunk 4]
```diff
@@ -1,57 +1,80 @@
...
 
-  const { setTabKey } = state!;
+  const { setTabKey } = state!
   const handleTabChange = (key: string) => {
-    dispatch!({ type: 'SET_SET_TAB_KEY', payload: { setTabKey: key } });
-  };
+    dispatch!({ type: 'SET_SET_TAB_KEY', payload: { setTabKey: key } })
+  }
+  let stp
+  if (state) {
```
Comment (fewhnhouse) [inline src/pages/build/Sets/Sets.tsx]: remove the if(state) and use const {bigPieceSelection, ...} = state!

mind the exclamation mark at the end

File: src/pages/build/Sets/Sets.tsx [hunk 5]
```diff
@@ -1,57 +1,80 @@
...
+    const {
+      bigPieceSelection,
+      smallPieceSelection,
+      jewelrySelection,
+      frontbarSelection,
+      backbarSelection
+    } = state
+
+    stp = bigPieceSelection
```
Comment (fewhnhouse) [inline src/pages/build/Sets/Sets.tsx]: rename to a more declarative variable name. make it const and remove the let declaration above.

File: src/components/GearCard.tsx [hunk 6]
```diff
@@ -45,146 +45,168 @@ const StyledTag = styled(Tag)`
...
+      <StyledTag color='#108ee9'>
+        {gear.selectedSet ? gear.selectedSet.type : 'No type'}
+      </StyledTag>
+      <Divider style={{ margin: '5px 0px' }} />
       <Description>
-        <div style={{ display: "flex", flexDirection: "column" }}>
-          {gear.selectedSet ? 
+        <div style={{ display: 'flex', flexDirection: 'column' }}>
+          {gear.selectedSet ? (
             totalBonus(gear.selectedSet).map(i => (
```
Comment (fewhnhouse) [inline src/components/GearCard.tsx]: rename variable to sth more descriptive, like count.

**Comment by khuusko** [comment]
Suggested edits committed

**Pull request closed without merging**